src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf

```diff
@@ -5247,7 +5247,7 @@ Ein catch()-Block nach einem catch (System.Exception e)-Block kann nicht-CLS-Aus
       </trans-unit>
       <trans-unit id="ERR_BadAsyncReturnExpression">
         <source>Since this is an async method, the return expression must be of type '{0}' rather than 'Task&lt;{0}&gt;'</source>
-        <target state="translated">Da es sich um eine asynchrone Methode handelt, muss der Rückgabeausdruck vom Typ "{0}" anstelle von "Task&lt;{0}&gt;" sein.</target>
+        <target state="translated">Der Rückgabetyp einer async-Methode muss "void", "Task", "Task&lt;T&gt;", ein taskähnlicher Typ oder "IAsyncEnumerable&lt;T&gt;" sein.</target>
```

Comment:
This translation was fine, but [`ERR_BadAsyncReturn`](https://github.com/dotnet/roslyn/pull/31327/files#diff-f7c7796cace976d0368b7b23a2e7469dR6418) still has the [wrong translation](https://github.com/dotnet/roslyn/pull/31327/files#r236001755).

Reply:
Please check this issue again. Fixed and updated PR.